docs/source/json_sheet_schema.rst

```diff
@@ -73,4 +73,44 @@ Step 4 is not implemented yet.
 On the one hand, custom fields allow for the definition of arbitrary "simple" values.
 Currently, it is possible to have boolean, numbers, strings, enums and lists of the atomic types.
 On the other hand, using JSON pointers, centrally defined field types can be used.
-This allows for easy sharing of data types and easier computat
+This allows for easy sharing of data types and easier computation.
+
+.. _predefined_fields:
```

Comment:
As stated earlier (?) I think the following section should not be part of the documentation for now.